runtimes/bundles/org.jboss.tools.rsp.server.wildfly/src/main/java/org/jboss/tools/rsp/server/wildfly/servertype/capabilities/Wildfly160ExtendedProperties.java

```diff
@@ -22,4 +24,9 @@ public Wildfly160ExtendedProperties(IServer obj) {
 	public String getRuntimeTypeVersionString() {
 		return "16.0"; //$NON-NLS-1$
 	}
+	@Override
+	public IDefaultLaunchArguments getDefaultLaunchArguments() {
+		return new Wildfly150DefaultLaunchArguments(server);
```

Comment:
why not have have Wildfly160ExtendedProperties extend ```Wildfly150ExtendedProperties``` instead of repeating yourself?

Reply:
sonar often complains if your heirarchy gets more than 4 or 5 subclasses deep. A few months back I removed a couple of the layers and reduplicated some small code to help make the code less of a 30-level-deep labyrinth.